e2e/pod.go

```diff
@@ -105,17 +105,40 @@ func waitForDeploymentComplete(name, ns string, c kubernetes.Interface, t int) e
 	return nil
 }
 
-func getCommandInPodOpts(f *framework.Framework, c, ns string, opt *metav1.ListOptions) (framework.ExecOptions, error) {
+func findPodAndContainerName(f *framework.Framework, c, ns, cn string, opt *metav1.ListOptions) (string, string, error) {
+	pn := ""
+	podList, err := f.PodClientNS(ns).List(context.TODO(), *opt)
+	framework.ExpectNoError(err)
+	if len(podList.Items) == 0 {
+		return "", "", errors.New("podlist is empty")
+	}
+	if cn != "" {
+		for i := range podList.Items {
+			for j := range podList.Items[i].Spec.Containers {
+				if podList.Items[i].Spec.Containers[j].Name == cn {
+					pn = podList.Items[i].Name
+					return pn, cn, nil
+				}
+			}
+		}
+		return "", "", errors.New("container name not found")
+	}
+	pn = podList.Items[0].Name
+	cn = podList.Items[0].Spec.Containers[0].Name
+	return pn, cn, nil
```

Comment:
just return podList.Items[0].Name and podList.Items[0].Spec.Containers[0].Name directly here and above

Reply:
Sure.